### ISSUE TITLE
version 0.11, Preview World Map freezes on State view mode after a few seconds

### ISSUE THREAD

**Issue by BobbertCanuck** [herbix/hoi4modutilities#81]
As it says on the tin, I open the World Map Preview and I set the view mode to State. 
After I hover over a few different states it locks up. 
It's not VS Code since everything else works fine. It's just the World map Preview that locks up. 
Such a shame too, I was really hyped when my buddy told me that 0.11 added rivers to the map preview.

**Comment by herbix** [comment]
I can't repro this on my side. Are you viewing the map from original game, or from any mod?
You can try close and reopen world map preview to see whether it works.
If it still exists, can you please paste the console logs from developer tools when this happens? Developer tools window can be opened from help menu.

![image](https://github.com/herbix/hoi4modutilities/assets/7110965/fee3113d-62d3-4616-9fe2-88a429ea1e3d)

**Comment by BobbertCanuck** [comment]
Hello, thanks for the speedy response.
It hadn't crossed my mind to check vanilla and sure enough it was working when I had vanilla's folder open.
The issue was only operating when I was using state view with the mod I was working on.

After checking the dev console as you suggested I found a few things in the console which led me to figure out what was causing the freeze. I will still send the console logs to you regardless at the end of this post.

I was lacking general_stuff.gfx and resource_strip.dds
After I added these files to their respective places it started to work as intended. We weren't planning on changing those files so we hadn't added them to our mod.

Strangely though, of the coders on the mod who've gotten a chance to try this, I was the only one who was having the freezing problem, The buddy I mentioned previously and I had a back-and-forth regarding the freeze and he said it was all working fine for him, so I find this just a  bit strange still.

anyways, here's the console log stuff that popped up when the map freezes were happening:
![console_log_i](https://github.com/herbix/hoi4modutilities/assets/82345168/5267fbd0-16e7-4225-b1ea-8b6b53e914c3)
![console_log_ii](https://github.com/herbix/hoi4modutilities/assets/82345168/411177f2-b5fa-4af8-a851-ee18cae40142)
![console_log_iii](https://github.com/herbix/hoi4modutilities/assets/82345168/6098bab9-5fb4-4fb1-9ba1-63d198957958)
![console_log_iv](https://github.com/herbix/hoi4modutilities/assets/82345168/b6ff397f-c8da-4f7e-b8d6-6f5653864eaf)

I'd be happy to close this but there's just something about the fact that my friend was having no issues where I was having issues that strikes me as odd.
